pages/common/ack.md

```diff
@@ -17,3 +17,11 @@
 - Show the file names containing "foo" and number of matches in each file:
 
 `ack -cl {{foo}}`
+
+- Search file "my_file" for string "foo bar":
```

Comment:
Search a given file for the term "foo". To maintain consistency with previous examples.